pysrc/SBProfile.cpp

```diff
@@ -79,6 +79,9 @@ namespace galsim {
                 .def(bp::self == bp::other<GSParams>())
                 .enable_pickling()
                 ;
+#if BOOST_VERSION >= 106000 && BOOST_VERSION < 106100
```

Comment:
Yeah, I used boost 1.61.0 for my python3 test.  If I leave the `< 106100` out, I get the same warning as for 1.59.5: 

```
RuntimeWarning: to-Python converter for boost::shared_ptr<galsim::GSParams> already registered; second conversion method ignored.
```

So I think this is just something that sneaked past the boost devs for a single release.  We could almost just ignore it, except that boost-1.60 is the version that ships with the LSST conda stack at the moment, so I have a feeling others may run into this like I did.